ui-modules/app-inspector/app/components/entity-tree/entity-node.html

```diff
@@ -35,15 +35,15 @@
     <div class="entity-node-children" ng-show="isChildrenOpen">
         <!-- Entity children -->
         <entity-node ng-repeat="child in entity.children track by child.id"
-                     ng-show="child.viewModes.has(viewMode)"
+                     ng-if="child.viewModes.has(viewMode)"
```

Comment:
Moved back ng-if since it re-refreshes the fade out animation for changed relationships after view switch.